spec/cc/engine/rubocop_spec.rb

```diff
@@ -239,6 +239,29 @@ def method
         assert !includes_check?(output, "Style/AndOr")
       end
 
+      it "ignores non-Ruby files even when passed in as include_paths" do
+        config_yml = "foo:\n  bar: \"baz\""
+        create_source_file("config.yml", config_yml)
+        output = run_engine(
+          "include_paths" => %w(config.yml)
```

Comment:
:lipstick: we prefer `%w[` for word arrays (note: you may have to tweak rubocop.yml like [this](https://github.com/codeclimate/builder/blob/master/.rubocop.yml#L44))